robottelo/cli/factory.py

```diff
@@ -3965,33 +3945,30 @@ def setup_cdn_and_custom_repos_content(
         repos=repos,
         download_policy=download_policy
     )
-    # Create a content view
```

Comment:
this saves us from a need of creating a custom LCE and promot custom CV to it if we don't need it